app/services/catalog/update_order_item.rb

```diff
@@ -1,3 +1,5 @@
+require "manageiq-messaging"
```

Comment:
Interesting, what brought this on?

Reply:
Specs were failing because it was being required in `lib/service_order_listener.rb` and now since that file doesn't exist, kaboom. I guess maybe it should be in the `update_order_item_spec.rb` instead, since that's what's technically building up the message, and the only place that's using the `UpdateOrderItem` class now is the internal API which just builds a struct.

I'll change it to move this to the spec file.